### ISSUE TITLE
[Power Up] Quantum CV RNN

### ISSUE THREAD

**Issue by AndreyYashkin** [XanaduAI/QHack2021#45]
### Team Name: 

QuantumMadness

### Project Description: 

This project is an attempt to build a quantum recurrent neural network using the continuous variable quantum model.  As sources of inspiration a [quantum qubit based RNN architecture](https://arxiv.org/pdf/2006.14619.pdf) which allows to reuse input wire, the quantum neural network [tutorial from strawberryfields.ai](https://strawberryfields.ai/photonics/demos/run_quantum_neural_network.html)  and the Elman RNN architecture were taken.

Due to the hight computational cost for the simulation of Fock space only primitive tests can be accomplished. RNN was tested to be suitable for learning XOR function from sequences of zeros and ones. From an investigation done so far it seems to be more sensitive to learning rate than its classic analog. Also the RNN circuit is quite long what may lead to a situation when a state created by the circuit is physically incorrect due to computational mistakes led by limited Fock dimensional size.

### Source code: 

[repo](https://github.com/nsu-ai/quantum-CV-RNN)

### Resource Estimate: 
We would like to get use a photonic CV quantum computer to test the proposed architecture on more complex tasks ML like the speech recognition which require RNNs with input and hidden sizes significantly bigger that 5 in summary.

**Comment by co9olguy** [comment]
Thanks for your Power Up Submission @AndreyYashkin !

To help us keep track of final submissions, we will be closing all of the [Power Up] issues. We ask you to open a new issue for your final submission. Please use [this pre-formatted [Entry] Issue template](https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=submission&template=open-hackathon-final-submission.md&title=%5BENTRY%5D+Your+Project+Title). Note that for the final submission, the Resource Estimate requirement is replaced by a Presentation item.
